packages/mixins/apollo-query-mixin.js

```diff
@@ -128,12 +128,6 @@ export const ApolloQueryMixin = superclass =>
        */
       this.errorPolicy = 'none';
 
-      /**
-       * Specifies the FetchPolicy to be used for this query.
-       * @type {FetchPolicy}
-       */
-      this.fetchPolicy = 'cache-first';
```

Comment:
This should be `this.fetchPolicy;` in order to maintain type safety via JSDoc. Tests should be added to ensure that the default passed to the client is respected.